### ISSUE TITLE
Add option to publish stubs

### ISSUE THREAD

**Issue by dpslwk** [geeksareforlife/laravel-doctrine-artisan#5]
Laravel 7 introduced new Stub Customisation
by publishing there stubs to /stubs/
could publish  to /stubs/vendor/... similar to how views are published

https://github.com/laravel/framework/commit/db4e5d3a5186b41bb0e1742381cc19ddc4ea86d3#diff-67cd0e59d939a533a834a6f4b204843b

**Comment by geeksareforlife** [comment]
good idea!  Relies on #4, but should be relatively easy.

It's a shame that we can't hook into the publish:stubs command...

**Comment by dpslwk** [comment]
Yeah standard pattern would be something like `vendor:publish —tag=stubs`
